docs/book/src/developer/providers/implementers-guide/controllers_and_reconciliation.md

```diff
@@ -90,7 +99,7 @@ func (r *MailgunClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	ctx := context.Background()
 	_ = r.Log.WithValues("mailguncluster", req.NamespacedName)
 
-	var cluster infrastructurev1alpha3.MailgunCluster
+	var cluster infrastructurev1beta1.MailgunCluster
```

Comment:
I wonder if we should switch those import aliases to `infrav1`.

In core CAPI we did that because when you bump the apiVersion you only have to change one import line and not your entire code.

Reply:
done